modules/cluster-logging-visualizer-kibana.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * logging/viewing/cluster-logging-visualizer.adoc
+
+[id="cluster-logging-visualizer-kibana_{context}"]
+= Viewing cluster logs in Kibana
+
+You can view cluster logs in Kibana. There are multiple ways to view and visualize your data that are beyond the scope of this documentation. For more information, refer to the link:https://www.elastic.co/guide/en/kibana/6.8/tutorial-sample-discover.html[Kibana documentation].
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+* Kibana index patterns must exist.
+
+* To list the *infra* and *audit* indices in Kibana, a user must have the `cluster-admin` role, the `cluster-reader` role, or both roles. The default `kubeadmin` user does not have proper permissions to list these indices. 
++
+If you can view the Pods and logs in the `default` project, you should be able to access the these indices. You can use the following command to check if the current user has proper permissions:
++
+[source,terminal]
+----
+$ oc auth can-i get pods/logs -n default
+----
++
+.Example output
+[source,terminal]
+----
+yes
+----
+
+.Procedure
+
+To view logs in Kibana:
+
+To launch Kibana:
+
+. In the {product-title} console, click the Application Launcher {launch} and select *Logging*.
+
+. Log in using the same credentials you use to log in to the {product-title} console.
++
+The Kibana interface launches.
+
+. In the Kibana, click *Discover*.
+
+. Select an index pattern: *app*, *audit*/*, or *infra*/*.
```

Comment:
Isn't that not entirely correct since you would usually have your own index pattern that you created when you access Kibana for the first time. @lukas-vlcek can you confirm?

Reply:
@sichvoge @lukas-vlcek There is a drop-down on the left side of the page where you can select between multiple index patterns, if you have more than one. 
I will make this more clear.

Reply:
@sichvoge  Yes, the users may have their own index pattern.   @mburke5678  How about:  select the index pattern you had created before. For example.   *app*, *audit*/*, or *infra*/*.